samples/PostCSS/sample.postcss

```diff
@@ -0,0 +1,13 @@
+@define-mixin size $size {
+  width: $size;
+}
+
+$big: 100px;
+
+/* Main block */
+.block {
+  &_logo {
+    background: inline("./logo.png");
+    @mixin size $big;
+  }
+}
```

Comment:
Where is this sample from? It looks very simple. Is it something you wrote specifically for this pull request?

We generally prefer more realistic samples as they are used to train the naive Bayes classifier.

Reply:
It's the existing postcss sample with the new extension. 😸

Reply:
Ah. Any chance you could find another sample from a "real" application under a permissive license?

Reply:
https://github.com/Akryum/awesomejs.dev/blob/e59a65f6d318cfad9f340681e843d2de20d032fb/packages/frontend/src/assets/styles/components/markdown.postcss#L1 (MIT)

Reply:
That looks fine :+1: